pkgs/games/tintin/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, lib, zlib, pcre
+, memorymappingHook, memstreamHook
+, tlsSupport ? true, gnutls
+# ^ set { tlsSupport = false; } to reduce closure size by ~= 18.6 MB
```

Comment:
First of all I would drop that number because it is probably outdated and secondly I am not sure if that is worth it at all. There is probably a gnutls on every system already.